### DISCUSSION TITLE
Добавление в группу меню из пакета

### DISCUSSION THREAD

**Discussion by ramir1** [moonshine-software/moonshine#1209] [Ideas]
Нужна возможность добавлять из пакета в группу меню.
Например,
```
MenuGroup::make('System')->setItems( [
                MenuItem::make('Link Project', 'https://moonshine-laravel.com/docs')
            ]);
```
в Пакете
```
MenuGroup::make('System')->setItems( [
                MenuItem::make('Link Project2', 'https://moonshine-laravel.com/docs')
            ]);
```
должен получить 1 пункт меню System с 2мя подпунктами

**Comment by lee-to** [comment]
https://moonshine-laravel.com/docs/resource/advanced/advanced-development#service-provider для пакетов vendorsMenu

**Reply by ramir1** [reply]
Это понятно. Речь о другом - добавлять в существующие группы меню пункты из пакета.
